plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/RepositoryComponent.groovy

```diff
@@ -168,16 +178,27 @@ extends DirectComponentSupport
   }
 
   @PackageScope
-  List<Repository> filter(final @Nullable StoreLoadParameters parameters) {
+  Iterable<Repository> filter(final @Nullable StoreLoadParameters parameters) {
     def repositories = repositoryManager.browse()
     if (parameters) {
       String format = parameters.getFilter('format')
       if (format) {
-        return repositories.findResults { Repository repository ->
-          repository.format == format
+        repositories = repositories.findResults { Repository repository ->
```

Comment:
Hmmm, forgot about this idiosyncrasy of findResults. Might be better off with just findAll which is a truer filter method